experiments/tsp/run_tsp.py

```diff
@@ -123,17 +125,20 @@
     # Define custom_model, config, and state based on GNN yes/no
     if args.use_gnn:
         custom_model = "TSPGNNModel"
-        custom_model_config = {"num_messages": 1, "embed_dim": 32}
+        custom_model_config = {"num_messages": 3, "embed_dim": 32}
```

Comment:
@davebiagioni, I think this was the most consequential change. we could even experiment with larger models if we scaled up the rollouts